recipes/pyophidia/meta.yaml

```diff
@@ -0,0 +1,35 @@
+package:
+    name: pyophidia
+    version: "1.4.0"
+
+source:
+    url: git://github.com/OphidiaBigData/PyOphidia/archives/v1.4.0.tar.gz
```

Comment:
this should be an `http` url not a `git` on.  Can you just use the following canonical url:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L18

it looks like it should be on pypi:
https://pypi.org/project/PyOphidia/#files

Note, you might consider using the jinja templating features as in:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L4-L19

just be aware that the name should be `PyOphidia` not `pyophidia` given the naming on pypi.